src/earn/EarnDepositBottomSheet.tsx

```diff
@@ -109,6 +114,9 @@ export default function EarnDepositBottomSheet({
         <Logos providerUrl={providerLogoUrl} />
         <Text style={styles.title}>{t('earnFlow.depositBottomSheet.title')}</Text>
         <Text style={styles.description}>{t('earnFlow.depositBottomSheet.description')}</Text>
+        <View style={styles.infoContainer}>
+          <EarnApyAndAmount tokenAmount={amount} token={token} testID={'EarnDepositBottomSheet'} />
```

Comment:
nit: rename testID -> testIDPrefix, this currently makes me thing this component has a testID called `EarnDepositBottomSheet`